src/infer_request.cc

```diff
@@ -442,6 +455,8 @@ InferenceRequest::RespondIfError(
           std::move(response), TRITONSERVER_RESPONSE_COMPLETE_FINAL, status),
       (request->LogRequest() + "failed to send error response").c_str());
 
+  request->ReportStatistics(request->model_raw_->MetricReporter().get(), false, 0, 0, 0, 0, reason);
```

Comment:
Add inline comments for what these hard-coded values are:
```cpp
request->ReportStatistics(..., false /* success */, 0 /* compute_start_ns */, ...)
```

Reply:
Add comments.